setup.cfg

```diff
@@ -28,11 +28,13 @@ python_requires = >=3.7
 setup_requires =
     setuptools_scm
 install_requires =
-    asdf>=2.7.1
+    # asdf 2.8.0 is unreleased so it needs to be installed from master:
+    asdf@git+https://github.com/asdf-format/asdf
```

Comment:
This will commit us to releasing `asdf 2.8` before we can do another official release of `jwst`.

Reply:
@nden do you think we'll have time to get that asdf release out?  I estimate I'll need about a week's worth of mopping up before the code is ready to go, plus some additional time for thorough testing.

Reply:
I'm going to replace this with an asdf 2.7 extension, since it's straightforward given that we only have schemas to add and no tags, and gives us one less thing to worry about.